### ISSUE TITLE
Ability to lump aspectual pairs together when checking target lemmas

### ISSUE THREAD

**Issue by arthurian** [Harvard-ATG/visualizing_russian_tools#4]
It should be possible to lump aspectual pairs together (e.g. начинать|начать) when checking target vocabulary. In other words, if начать (perfective) is listed in the target vocabulary, but a form of начинать (imperfective) is used in the story text, then it should be marked as used.

## Scenario 1

Input Story: 
> Я начинаю урок завтра.

Input Vocabulary:
> начать
> завтра

| Actual Output          | Desired Output        |
|------------------------|------------------------|
| Я начинаю урок **завтра**. | Я **начинаю** урок **завтра**. |

The word _начинаю_ is used in the story, and has lemma начинать (imperfective). But the vocabulary only lists начать (perfective). It should be recognized that начинать|начать form an aspectual pair, and therefore _начинаю_ should be marked in the story as used.

## Scenario 2

Input Story:
> Я начал учиться вчера.

Input Vocabulary:
> начинать
> вчера

| Actual Output          | Desired Output        |
|------------------------|------------------------|
| Я начал учиться **вчера**. | Я **начал** учиться **вчера**.|

The word _начал_ is used in the story, and has lemma начать (perfective). But the vocabulary only lists начинать (imperfective). It should be recognized that начинать|начать form an aspectual pair, and therefore _начал_ should be marked in the story as used.

**Comment by stevenclancy** [comment]
Hi Artie, Is it helpful if I comment here (more helpful than in emails?)?

**Comment by stevenclancy** [comment]
Yes, the above is a good description. I could see the desire by some users NOT to lump together the aspectual pairs and just to deal with the lemmas for the verbs, as in начинать, начать as individual words, but most of the time I think most teachers and students should be considering these as parts of a combined lemma.

**Comment by arthurian** [comment]
@stevenclancy This is probably most helpful for following up on specific issues/functionality, but generally I think email is probably best, at least for initial discussion and feedback.

Ok, so the behavior should be implemented as the default (treat them as parts of a combined lemma), with the option to switch it off, should the user want to treat the verbs as individual words.
